src/VerificationCodeManager.php

```diff
@@ -43,10 +43,11 @@ public function send(string $verifiable, string $channel = 'mail')
      *
      * @param string $code
      * @param string $verifiable
+     * @param bool $cleanup
      *
      * @return bool
      */
-    public function verify(string $code, string $verifiable)
+    public function verify(string $code, string $verifiable, bool $cleanup = true)
```

Comment:
On second thought, maybe we should call it `$deleteAfterVerification` 🤔 so there can not be any confusion about the meaning.

can you also document this extra option?